doc/guides/collaborator-guide.md

```diff
@@ -120,14 +120,22 @@ needed [approvals](#code-reviews), [CI](#testing-and-ci), and
 except the [wait time](#waiting-for-approvals), please add the
 [`author ready`](#author-ready-pull-requests) label.
 
-Where there is disagreement among Collaborators, consensus should be sought if
-possible. If reaching consensus is not possible, a Collaborator may escalate the
-issue to the TSC.
-
-Collaborators should not block a pull request without providing a reason.
-Another Collaborator may ask an objecting Collaborator to explain their
-objection. If the objector is unresponsive, another Collaborator may dismiss the
-objection.
+If a collaborator believes a pull request should not land as is, **the "Request
+Changes" GitHub feature must be used to make the objection explicit**. An
+implicit objection not using the "Request Changes" feature is not considered a
```

Comment:
Since the goal here is to make things as clear as possible, I'd recommend:

```suggestion
implicit objection not using the "Request Changes" feature is not a
```